### DISCUSSION TITLE
Is there a way to adapt this to an A10s based MK802+ clone?

### DISCUSSION THREAD

**Discussion by Drevi** [johang/sd-card-images#51] [Q&A]
Hello, 
I have and old HDMI stick that I would like to run linux on. It's a generic clone of the MK802+. I've opened it and have pictures of every chip to complement the little info that is available in the device details of android. 
I tried the only image for A10s based board ([A10s OLinuXino MICRO](https://sd-card-images.johang.se/boards/a10s_olinuxino_micro.html)) and it did not work. 
I've also tried using MK802 and MK802II images, but as suspected it didn't work as they're based on A10 and not A10S. 

Is there a way to adapt this image to work in this particular stick?

**Comment by johang** [comment]
I build images for boards that are supported in mainline U-Boot and mainline Linux kernel. I can't do special things for special boards that are not in mainline U-Boot and mainline Linux kernel.
